nixos/modules/services/desktops/pipewire/easyeffects.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.pipewire.easyeffects;
+in
+{
+  meta.maintainers = with lib.maintainers; [ starcraft66 ];
+
+  options = {
+    services.pipewire.easyeffects = {
+      enable = lib.mkEnableOption "Audio effects for PipeWire applications.";
+
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.easyeffects;
+        defaultText = lib.literalExpression "pkgs.easyeffects";
+        description = ''
+          The easyeffects derivation to use.
+        '';
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = config.services.pipewire.enable && !config.hardware.pulseaudio.enable;
+        message = "Easyeffects requires Pipewire to be enabled and is not compatible with PulseAudio.";
+      }
+    ];
+
+    environment.systemPackages = [ cfg.package ];
+    systemd.packages = [ cfg.package ];
```

Comment:
There is nothing for systemd to install in the package.

Reply:
There is a dbus service file in `share/dbus-1/services/`. I'm guessing putting `pkgs.easyeffects` in `services.dbus.packages` does the same thing as this entire module?

Reply:
D-Bus services would not be picked up by `services.dbus.packages`. And they are already picked  by `environment.systemPackages`. As I understand it, the point of this module is to have EasyEffects autostart, which can be easily achieved by enabling autostart in settings, which will create the following file in `~/.config/autostart/easyeffects-service.desktop`:

```ini
[Desktop Entry]
Name=EasyEffects
Comment=EasyEffects Service
Exec=easyeffects --gapplication-service
Icon=easyeffects
StartupNotify=false
Terminal=false
Type=Application
```

Reply:
> As I understand it, the point of this module is to have EasyEffects autostart

That is precisely why I decided to create this module. I think it is a much more elegant solution rather than relying on the kind of stateful workaround you are suggesting to start the "service". Additionally, just launching a one-off process like that makes it more annoying to restart the service etc.